src/til/ut_til/string.cpp

```diff
@@ -53,6 +53,56 @@ class StringTests
         VERIFY_IS_TRUE(til::ends_with("0abc", "abc"));
     }
 
+    // Normally this would be the spot where you'd find a TEST_METHOD(from_wchars).
+    // I didn't quite trust my coding skills and thus opted to use fuzz-testing.
+    // The below function was used to test from_wchars for unsafety and conformance with clang's strtoul.
+    // The test was run as:
+    //   clang++ -fsanitize=address,undefined,fuzzer -std=c++17 file.cpp
+    // and was run for 20min across 16 jobs in parallel.

```

Comment:
We may want to file a backlog follow-up to add this test to onefuzz after #10431 merges